### PR TITLE
PR template update

### PR DESCRIPTION
**Pull request by abottoms-coder**
### Summary of Changes
_Provide a brief summary of changes_
_Addresses issue #808
 - [ ] PR Template requests GIFs/screenshots
 - [ ]  PR Template enforces contextual comments
 - [ ]  Update How to test section
 - [ ]  Preliminary QASP checklist outlines deployment process
 - [ ]  Testing Checklist has been run and all tests pass
 - [ ]  README is updated, if necessary

### How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/ and sign in.
1. Proceed with functional tests as described herein.
1. Test steps should be captured in the demo GIF(s) and/or screenshots below.

> *Demo GIF(s) and screenshots for testing procedure*

### Deliverable 1: Accepted Features

> *Performance Standard(s): At the beginning of each sprint, the Product Owner and development team will collaborate to define a set of user stories to be completed during the sprint. Acceptance criteria for each story will also be defined. The development team will deliver code and functionality to satisfy these user stories.*

> *Acceptable Quality Level: Delivered code meets the acceptance criteria for each user story. Incomplete stories will be assessed and considered for inclusion in the next sprint.*

+ [ ] Look up the acceptance criteria in the related issue; paste ACs below in checklist format.
+ [ ] Check against the criteria:

As Product Owner, @lfrohlich will decide if ACs are met.

### Deliverable 2: Tested Code

> *Performance Standard(s): Code delivered under the order must have substantial test code coverage. Version-controlled HHS GitHub repository of code that comprises products that will remain in the government domain.*

> *Acceptable Quality Level: Minimum of 90% test coverage of all code. All areas of code are meaningfully tested.*

+ Are all areas of code introduced in this PR meaningfully tested?
  + [ ] If this PR introduces backend code changes, are they meaningfully tested?
  + [ ] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [ ] Frontend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)
  + [ ] Backend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)

### Deliverable 3: Properly Styled Code

> *Performance Standard(s): GSA 18F Front- End Guide*

> *Acceptable Quality Level: 0 linting errors and 0 warnings*

+ [ ] Are backend code style checks passing on CircleCI?
+ [ ] Are frontend code style checks passing on CircleCI?
+ [ ] Does this PR change any linting or CI settings?

### Deliverable 4: Accessible

> *Performance Standard(s): Web Content Accessibility Guidelines 2.1 AA standards*

> *Acceptable Quality Level: 0 errors reported using an automated scanner and 0 errors reported in manual testing*
+ [ ] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with @iamjolly and @ttran-hub using Accessibility Insights reveal any errors introduced in this PR?
    + [See the full Accessibility Assessment plan here.](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)

### Deliverable 5: Deployed

> *Performance Standard(s): Code must successfully build and deploy into the staging environment.*

> *Acceptable Quality Level: Successful build with a single command*

+ [ ] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### Deliverable 6: Documented

> *Performance Standard(s): Summary of user stories completed every two weeks. All dependencies are listed and the licenses are documented. Major functionality in the software/source code is documented, including system diagram. Individual methods are documented inline in a format that permits the use of tools such as JSDoc. All non-inherited 800-53 system security controls are documented in the Open Control or OSCAL format and HHS Section 508 Product Assessment Template (PAT) are updated as appropriate.*

> *Acceptable Quality Level: Combination of manual review and automated testing, if available*

+ [ ] If this PR introduces backend code, is that code documented both inline and overall?
+ [ ] If this PR introduces frontend code, is that code documented both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?

### Deliverable 7: Secure

> *Performance Standard(s): Open Web Application Security Project (OWASP) Application Security Verification Standard 3.0*

> *Acceptable Quality Level: Code submitted must be free of medium- and high-level static and dynamic security vulnerabilities*

+ [ ] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any security issues?

### Deliverable 8: Context

> * Performance Standard(s): Code must be understandable and contextualized for the reviewers possess the knowledge and background necessary for analysis and constructive criticism to take place.

> * Acceptable Quality Level: Code submitted in the pull request has context.*
+ [ ] Does this pull request contain sufficient inline comments providing relevant context for code snippets?
+ [ ] Is the code understandable and lucid?
+ [ ] Does this pull request provide background for why coding decisions were made?
+ [ ] Can you as a reviewer explain and take ownership of these elements presented in this code review?

### CODE REVIEW

**Comment by abottoms-coder** [comment]
@ADPennington @amilash @lfrohlich 

Would like your feedback on if this new template covers or is approaching the ACs.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -55,8 +68,6 @@ As facilitator/product manager, @kniz-raft will decide if ACs are met from Raft'
 
 > *Acceptable Quality Level: Successful build with a single command*
```
Comment (jtwillis92) [inline .github/pull_request_template.md:69]:
```suggestion
> *Acceptable Quality Level: Successful deployment by assigning a `Deploy with CircleCI - {env}` label*
```

**Review by jtwillis92 — APPROVED** [review]
Left a minor suggestion but otherwise LGTM

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -45,17 +56,17 @@ As facilitator/product manager, @kniz-raft will decide if ACs are met from Raft'
 > *Performance Standard(s): Web Content Accessibility Guidelines 2.1 AA standards*
 
 > *Acceptable Quality Level: 0 errors reported using an automated scanner and 0 errors reported in manual testing*
-
++ [ ] Does this PR complete the epic? 
++ [ ] Are links included to any other gov-approved PRs associated with epic?
++ [ ] Does PR include documentation for Raft's a11y review? 
 + [ ] Did automated and manual testing with @iamjolly and @ttran-hub using Accessibility Insights reveal any errors introduced in this PR?
     + [See the full Accessibility Assessment plan here.](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
```
Comment (ADPennington) [inline .github/pull_request_template.md:63]:
```suggestion
    + [See the full Accessibility Assessment plan here.](https://github.com/HHS/TANF-app/blob/main/docs/Technical-Documentation/how-government-will-test-a11y.md)
```

**Review by ADPennington — CHANGES_REQUESTED** [review]
looks good @abottoms-coder . one minor fix requested.

File: .github/pull_request_template.md (hunk 2 above)
Comment (abottoms-coder) [reply .github/pull_request_template.md]: Good catch!

**Review by ADPennington — APPROVED** [review]
lgtm 👍🏾 ready to merge @abottoms-coder

**Pull request merged**